Purchases/Caching/DeviceCache.swift

```diff
@@ -493,4 +499,38 @@ fileprivate extension UserDefaults {
 
 }
 
+private extension DeviceCache {
+    enum CacheDuration {
+        // swiftlint:disable:next nesting
+        enum AppStatus {
```

Comment:
Love the nested types 😄  Swift is the best.

Reply:
💯